Chap_API_Tools.tex

```diff
@@ -749,10 +749,12 @@ \subsection{Co-Location of Debugger Daemons}
 given node are responsible for self-determining their specific target
 process(es) - e.g., by referencing their own \refattr{PMIX_LOCAL_RANK} in the
 daemon debugger job versus the corresponding \refattr{PMIX_LOCAL_RANK} of the
-target processes on the node. Note that this method for starting the debugger
-precludes the use of \ac{PMIx} attributes to \emph{pause} the application prior
-to attaching the debugger. Users are referred to the \refapi{PMIx_Job_control}
-\ac{API} for possible alternative methods.
+target processes on the node. Note that this method for starting the debugger daemons
+precludes the use of \ac{PMIx} attributes to \emph{pause} the target processes prior
+to attaching the debugger.
+Note that \ac{PMIx} job \emph{pause} attributes (e.g., \refattr{PMIX_DEBUG_STOP_IN_INIT}) do not apply to applications (defined in \refstruct{pmix_app_t}) where the \refattr{PMIX_DEBUGGER_DAEMONS} attribute is set to \code{true}.
+Users are referred to the \refapi{PMIx_Job_control}
+\ac{API} for possible alternative methods for pausing a job already in execution.
```

Comment:
Wait a minute - I got all turned around here by this blasted diff display. I thought you were addressing the "co-spawn" section. I'll address that separately as github won't let me do it here (since you didn't change it).

In this context, the application is already running - so of course you can't use the job attributes to pause it. You aren't launching the application - it is already executing, and you are simply attaching to it. Perhaps the better clarification would be:

```suggestion
target processes on the node. Note that the debugger will be attaching to the application processes
at some arbitrary point in the application's execution unless some method for pausing the application
(e.g., by providing a \ac{PMIx} directive at time of launch, or via a tool using the
\refapi{PMIx_Job_control} \ac{API} to direct that the process be paused) has been employed.
```

Reply:
That wording does clarify it. I can move the prior wording down to the co-spawn case in this PR.